### PR TITLE
Translation: Update some variable names

### PR DESCRIPTION
**Pull request by Dirain1700**
Deleted these lines:
```diff
- "User ${name} must be in the battle room already.": "",
- "${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
```

fixed this line:
```diff
- "${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+ "${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```

ref: #8303 

I couldn't catch all changes; please tell me :)

### CODE REVIEW

File: translations/english/core-commands.ts [hunk 1]
```diff
@@ -88,14 +88,13 @@ export const translations: Translations = {
...
 		"${targetUser.name} is already a player in this battle.": "",
-		"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
 		"Player 2": "",
 		"Players could not be restored (maybe this battle already has two players?).": "",
 		"This game doesn't support /joingame": "",
 		"This game doesn't support /leavegame": "",
 		"You can only do this in unrated non-tour battles.": "",
 		"User ${targetUsername} not found.": "",
-		"${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+		"${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```
Comment (AnnikaCodes) [inline translations/english/core-commands.ts:97]: It doesn't matter what the variables are called in translation. `displayReason` is clearer, though, so I'm fine with it.

Comment (Dirain1700) [reply translations/english/core-commands.ts:97]: Oh, Thank you!
(I'm just modifying it to reflect the changes)

Comment (Dirain1700) [reply translations/english/core-commands.ts:97]: I'm just modifying it to reflect the changes :)

**Comment by Slayer95** [comment]
> I couldn't catch all changes; please tell me :)

Is anybody willing to collaborate? If not, this surely can be merged as-is as a follow-up to the linked refactor PR.

**Comment by Dirain1700** [comment]
Please use this in commit message:

```txt
Deleted these lines:
"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
"User ${name} must be in the battle room already.": "",
"This room already has a player in slot ${target}.": "",
"${targetUser.name} is already a player in this battle.": "",
"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",

fixed this line:
- "${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+ "${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",

ref: #8303
```

File: translations/english/core-commands.ts [hunk 2]
```diff
@@ -83,19 +83,14 @@ export const translations: Translations = {
 		"${user.name} hid the replay of this battle.": "",
 		"You can only do this in battle rooms.": "",
 		"You can only add a Player to unrated battles.": "",
-		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
```
Comment (AnnikaCodes) [inline translations/english/core-commands.ts:86]: This translation seems to be still in use: https://github.com/smogon/pokemon-showdown/blob/0475b38ae8ad2cd1ea0b2849972a9c821bd6a1de/server/chat-commands/core.ts#L952

File: translations/english/core-commands.ts (hunk 1 above)
Comment (AnnikaCodes) [reply translations/english/core-commands.ts:93]: It... doesn't need to reflect the changes, but this is fine either way.

File: translations/english/core-commands.ts [hunk 3]
```diff
@@ -85,17 +85,13 @@ export const translations: Translations = {
 		"You can only add a Player to unrated battles.": "",
 		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
 		"User ${name} not found.": "",
-		"User ${name} must be in the battle room already.": "",
-		"This room already has a player in slot ${target}.": "",
```
Comment (AnnikaCodes) [inline translations/english/core-commands.ts:89]: This translation is also still in use: https://github.com/smogon/pokemon-showdown/blob/5105cc46a377e3effbb3cf5afb9b0ee826b29593/server/chat-commands/core.ts#L968

File: translations/english/core-commands.ts [hunk 4]
```diff
@@ -85,17 +85,13 @@ export const translations: Translations = {
...
-		"${targetUser.name} is already a player in this battle.": "",
```
Comment (AnnikaCodes) [inline translations/english/core-commands.ts:90]: As is this one: https://github.com/smogon/pokemon-showdown/blob/5105cc46a377e3effbb3cf5afb9b0ee826b29593/server/chat-commands/core.ts#L976

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**